code/modules/client/preference_setup/loadout/lists/suits.dm

```diff
@@ -4,34 +4,42 @@
 	path = /obj/item/clothing/suit/apron
 	slot = slot_wear_suit
 	sort_category = "Suits and Overwear"
-	cost = 1
 
 /datum/gear/suit/jacket
 	display_name = "jacket"
 	path = /obj/item/clothing/suit/storage/toggle/bomber
-	cost = 2 //higher price because it has some armor value or style
+	cost = 1 //higher price because it has some armor value or style (old)
+	//I'm dropping the price because the style is the same either way
+	//and a few extra points in armor roundstart is useless 5 minutes into the round anyways
```

Comment:
Perhaps we should remove the armor values?

Reply:
I mean maybe? But it's 1-3 points, who cares honestly? It becomes irrelevant 5 minutes in and there's really no point to worry about it.

Reply:
Powergamers care. 
Alas we have a lot of those in our playerbase.

Reply:
Alright, I wouldn't be against removing the armor
doesn't really make a difference, but alright...